packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -838,22 +838,21 @@ export default class New extends Component {
                       />
                     </fieldset>
                   )}
-                  {type === 'lvm' ||
-                    (type === 'ext' && (
-                      <fieldset>
-                        <label htmlFor='srDevice'>{_('newSrDevice')}</label>
-                        <input
-                          id='srDevice'
-                          className='form-control'
-                          placeholder={formatMessage(
-                            messages.newSrLvmDevicePlaceHolder
-                          )}
-                          ref='device'
-                          required
-                          type='text'
-                        />
-                      </fieldset>
-                    ))}
+                  {(type === 'lvm' || type === 'ext') && (
```

Comment:
@olivierlambert Do you agree with my change here?

Reply:
That's exactly how I wrote it before it was automatically reformatted…

Reply:
Probably not, because this has also been automatically formatted :slightly_smiling_face:

Reply:
I wrote it like this (I copy/pasted you suggestion you gave me previously). I have no logical explanation but I'm 99% sure it was like that.